storm-server/src/main/java/org/apache/storm/daemon/nimbus/Nimbus.java

```diff
@@ -3907,6 +3938,12 @@ public void deleteBlob(String key) throws AuthorizationException, KeyNotFoundExc
                     throw new WrappedIllegalStateException(message);
                 }
             }
+            String topoId = topologyUsingThisBlob(stormClusterState, topoCache,  key);
+            if (null != topoId) {
```

Comment:
In terms of style null == something or null != something seems dated to when these mistakes (a = null) instead of (a == null) would not get caught by IDE's. But it is harder to read, not necessary anymore, and counter to style elsewhere in the code.

Reply:
done